src/User32/User32.Helpers.cs

```diff
@@ -369,5 +369,93 @@ public static unsafe WINDOWPLACEMENT GetWindowPlacement(IntPtr hWnd)
 
             return result;
         }
+
+        /// <summary>
+        /// Changes an attribute of the specified window. The function also sets a value at the specified
+        /// offset in the extra window memory.
+        /// </summary>
+        /// <param name="hWnd">A handle to the window and, indirectly, the class to which the window belongs.
+        /// The SetWindowLongPtr function fails if the process that owns the window specified by the
+        /// <paramref name="hWnd"/> parameter is at a higher process privilege in the UIPI hierarchy than the
+        /// process the calling thread resides in.</param>
+        /// <param name="nIndex">The zero-based offset to the value to be set. Valid values are in the range zero
+        /// through the number of bytes of extra window memory, minus the size of a LONG_PTR. To set any other value,
+        /// specify one of the following values.
+        ///
+        /// <list type="table">
```

Comment:
nice translation to xml! 😄

Reply:
Thanks, didn't know until now that "list" could mean "table" :)